docs/opentelemetry.rst

```diff
@@ -0,0 +1,102 @@
+OpenTelemetry: Monitoring Performance
+=====================================
+
+What is OpenTelemetry?
+----------------------
+
+`OpenTelemetry <https://opentelemetry.io>`_ is an open-source observability framework for `distributed traces <https://uptrace.dev/opentelemetry/distributed-tracing.html>`_, `metrics <https://uptrace.dev/opentelemetry/metrics.html>`_, and logs.
+
+OpenTelemetry allows developers to collect and export telemetry data in a vendor agnostic way. With OpenTelemetry, you can instrument your application once and then add or change vendors without changing the instrumentation, for example, here is a list of `popular DataDog alternatives <https://uptrace.dev/get/compare/datadog-competitors.html>`_ that support OpenTelemetry.
+
+OpenTelemetry instrumentation
+-----------------------------
+
+Instrumentations are plugins for popular frameworks and libraries that use OpenTelemetry API to record important operations, for example, HTTP requests, DB queries, logs, errors, and more.
+
+To install OpenTelemetry `instrumentation <https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/redis/redis.html>`_ for redis-py:
+
+.. code-block:: shell
+
+   pip install opentelemetry-instrumentation-redis==0.35b0
```

Comment:
IMHO if we tie this to a version, it's one more thing to maintain.

Reply:
Agreed, it is a copy-paste mistake